src/components/templates/Embeddable/Embeddable.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+import { EmbeddableVenue, Venue } from "types/venues";
+
+import "./Embeddable.scss";
+
+export interface EmbeddableProps {
+  venue: Venue;
+}
+
+export const Embeddable: React.FC<EmbeddableProps> = ({ venue }) => {
+  const typedVenue = venue as EmbeddableVenue;
+
```

Comment:
It's probably better to force the type to be correct rather than hack it later?

```suggestion
export interface EmbeddableProps {
  venue: EmbeddableVenue;
}

export const Embeddable: React.FC<EmbeddableProps> = ({ venue }) => {
```

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1243/commits/7c16f06cbda604a7b2357c9ef4c6c6f5d282eddb (enabled by https://github.com/sparkletown/sparkle/pull/1243/commits/fdb720c9da663e8dd68aecfefbbd56cb1901dba0 + https://github.com/sparkletown/sparkle/pull/1243/commits/03a9443ecca1f7f93f4744296feadd54cb2ea4e7)